db/seeds.rb

```diff
@@ -1,8 +1,6 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+GeneralInfoItem.create!(
```

Comment:
maybe you have unexpected errors if you run the seed twice. take a look at  the `find_or_create_by ` method

Reply:
Nice!